cartographer/io/color.cc

```diff
@@ -69,6 +68,7 @@ Color GetColor(int id) {
   CHECK_GE(id, 0);
   // Uniform color sampling using the golden ratio from
   // http://martin.ankerl.com/2009/12/09/how-to-create-random-colors-programmatically/
+  float kGoldenRatioConjugate = (std::sqrt(5.f) - 1.f) / 2.f;
```

Comment:
Or even better just use the constant provided by boost:
http://www.boost.org/doc/libs/1_63_0/libs/math/doc/html/math_toolkit/constants.html